<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QMainWindow" name="CreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить пользователя</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QMainWindow #centralwidget{
background-color:#0D1B2A;
}

.QPushButton {
	background-color:#293241;
	color:#E0E1DD;
}

.QRadioButton{
color:#E0E1DD;
}
#statusbar{
background-color:#293241;
color:#E0E1DD;
}

#menubar{
color:black;
font-weight:bold;
}

.QGroupBox{
	color:#E0E1DD
}

.QPushButton:hover{
	background-color:#1b263b;
}

#statusbar{
	color:#E0E1DD;
}
.QLabel{
	color:#e0e1dd;
	font-weight:bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>240</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Тип учетной записи</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="User">
            <property name="text">
             <string>Пользователь</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Admin">
            <property name="text">
             <string>Администратор</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Логин:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_Login"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Пароль:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_Password"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_CreateUser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
